# DRIVE=1,
# DRIVE_HIGH=4,
# LOAD_CMD=7,
# FRONT=10,
# HIGH=14,
# STAIRS=17,
# LOW=21,
# PLACE_CMD=25,
# STOW=29,
# STAND=32,
# BOX=35,
# FIBER=39,
# SHELL_WATCH=42,
# SHELL_LOAD_READY=46,
# SHELL_LOAD=52,
# SHELL_LIFT=58
# arm pose
# the value has to be one of the above
uint32 arm_pose
